src/passes/pass.cpp

```diff
@@ -311,7 +311,9 @@ void PassRegistry::registerPasses() {
                createOptimizeInstructionsPass);
   registerPass(
     "optimize-stack-ir", "optimize Stack IR", createOptimizeStackIRPass);
+#ifndef __EMSCRIPTEN__
```

Comment:
It would be good to add comment both here and on the other use of __EMSCRIPTEN__ to say why it is necessary.

Reply:
done